packages/react-components/react-drawer/stories/Drawer/DrawerCustomSize.stories.tsx

```diff
@@ -21,7 +22,7 @@ export const CustomSize = () => {
   const inputId = useId('custom-size-label');
 
   const [open, setOpen] = React.useState(false);
-  const [customSize, setCustomSize] = React.useState(250);
+  const [customSize, setCustomSize] = React.useState(600);
```

Comment:
I know it's already in master, so you can address this in a follow up, but we really shouldn't be encouraging users to use inline styles to set custom size, when this can easily be done (and should be done) using Griffel

Reply:
The intent of this example was to have a dynamic size controlled by the input.
As Griffel cannot create dynamic styles the only choice was to use inline styles. The only possible "fix" here would be creating an example with "fixed" custom sizes, which is quite limiting to the experience. Any ideas?